modules/logging-architecture-overview.adoc

```diff
@@ -24,6 +34,4 @@ The {logging-title} collects container logs and node logs. These are categorized
 
 * `audit` - Logs generated by auditd, the node audit system, which are stored in the */var/log/audit/audit.log* file, and logs from the `auditd`, `kube-apiserver`, `openshift-apiserver` projects, as well as the `ovn` project if enabled.
 
-The logging collector is a daemonset that deploys pods to each {product-title} node. System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}.
-
-Container logs are generated by containers running in pods running on the cluster. Each container generates a separate log stream. The collector collects the logs from these sources and forwards them internally or externally as configured in the `ClusterLogForwarder` custom resource.
+System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}.
```

Comment:
```suggestion
Journald log messages from the operating system, container runtime, and {product-title} generate system and infrastructure logs.
```

Reply:
not sure if the technical meaning changes here.

Reply:
Updated and removed this section anyway